packages/installer/image-mtree-check.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+exec >> /tmp/image-mtree-check.log
+exec 2>&1
+event="$1"
+
+if [ "$event" == "package.install" ]; then
+  payload="$2"
+  file=$(echo "$payload" | jq -r .file)
+
+  version=$(jq -r .Package.version < "$file")
+  repo=$(jq -r .Repository.urls[0] < "$file")
+  name=$(jq -r .Package.name < "$file")
+  category=$(jq -r .Package.category < "$file")
+  download_version=$(echo "$version"|tr "+" "-")
+
+  image="$repo":"$name"-"$category"-"$download_version".metadata.yaml
+  tmpdir=/tmp/"$name"-"$category"-"$download_version"-metadata
+  mtree_output=/tmp/"$name"-"$category"-"$download_version".mtree
+
+  echo "Getting $image metadata"
+
+  luet util unpack "$image" "$tmpdir"
+  # This is terrible, if this works we need to move to a golang binary FAST
+  curl https://github.com/mikefarah/yq/releases/download/v3.4.1/yq_linux_amd64 -L --silent --output /tmp/yq && chmod +x /tmp/yq
```

Comment:
Maybe we can add yq in the toolchain dir for the moment as we do for the other components, wdyt?

Reply:
Im struggling with that, like Its the way to go, but adding a package to the "base" to then remove it...I dont really like it. But I think its the way to go until we move to the single binary/plugin/whatever

Reply:
yep.. at the moment we have no much choice, but it's fine for the moment :) we have issues to track this :t-rex: